upgrade_test.py

```diff
@@ -664,10 +675,16 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
         self.metric_has_data(
             metric_query='sct_cassandra_stress_write_gauge{type="ops", keyspace="keyspace1"}', n=5)
 
-        # start gemini write workload
-        if self.version_cdc_support():
```

Comment:
you wrote you gonna remove it in a follow PR, but started removing usage of it here...

Reply:
removed this method at all. all latest supported scylla versions supports the cdc.